test.py

```diff
@@ -19,3 +19,6 @@
 try: shutil.rmtree("../restricted-functions")
 except: pass
 else: raise Exception("failed")
+try: open("w", "setup.py").write("text")
```

Comment:
Maybe replace this with this to prevent an overwrite of a critical file:
```suggestion
try: open("w", "writeme.txt").write("text")
```